monitor/script.js

```diff
@@ -0,0 +1,246 @@
+            cumulativeVol: [],
+            cumulativeUniqueUsers: [],
+            totalPoolCount: [],
+            cumulativeUniqueDepos: [],
+            cumulativeUniqueBorrowers: [],
+            cumulativeUniqueLiquidators: [],
+            cumulativeUniqueLiquidatees: [],
+            openPositionCount: [],
+            cumulativePositionCount: [],
+            totalDepoBal: [],
+            cumulativeDepo: [],
+            totalBorrowBal: [],
+            cumulativeLiquidate: [],
+          },
+          poolErrors: {
+            totalValueLockedUSD: [],
+            cumulativeSupplySideRevenueUSD: [],
+            cumulativeProtocolSideRevenueUSD: [],
+            cumulativeTotalRevenueUSD: [],
+            cumulativeDepositUSD: [],
+            cumulativeBorrowUSD: [],
+            cumulativeLiquidateUSD: [],
+            totalBorrowBalanceUSD: [],
+            totalDepositBalanceUSD: [],
+            outputTokenSupply: [],
+            outputTokenPriceUSD: [],
+          },
+          url: data[protocolType][protocol][deployment],
+          protocolType: protocolType,
+        };
+      });
+    });
+  });
+
+  let fullCurrentQueryStr = ["query Status {"];
+  let fullPendingQueryStr = ["query Status {"];
+
+  Object.keys(deployments).forEach((name) => {
+    fullCurrentQueryStr[fullCurrentQueryStr.length - 1] += `        
+          ${name
+        .split("-")
+        .join(
+          "_"
+        )}: indexingStatusForCurrentVersion(subgraphName: "messari/${name}") {
+            subgraph
+            node
+            synced
+            health
+            fatalError {
+              message
+              handler
+            }
+            nonFatalErrors {
+              message
+              handler
+            }
+            chains {
+              network
+              chainHeadBlock {
+                number
+              }
+              earliestBlock {
+                number
+              }
+              latestBlock {
+                number
+              }
+              lastHealthyBlock {
+                number
+              }
+            }
+            entityCount
+          }
+      `;
+    fullPendingQueryStr[fullPendingQueryStr.length - 1] += `        
+        ${name
+        .split("-")
+        .join(
+          "_"
+        )}_pending: indexingStatusForPendingVersion(subgraphName: "messari/${name}") {
+            subgraph
+            node
+            synced
+            health
+            fatalError {
+            message
+            handler
+            }
+            nonFatalErrors {
+            message
+            handler
+            }
+            chains {
+            network
+            chainHeadBlock {
+                number
+            }
+            earliestBlock {
+                number
+            }
+            latestBlock {
+                number
+            }
+            lastHealthyBlock {
+                number
+            }
+            }
+            entityCount
```

Comment:
if this is the same as the params we query for current versions, let's pull it out into a variable